mpact/admin.py

```diff
@@ -39,3 +42,93 @@ class UserAdmin(BaseUserAdmin):
         ),
     )
     inlines = (UserProfileInline,)
+
+
+class CustomPeriodicTask(PeriodicTask):
+    message = models.TextField(
+        blank=True,
+        verbose_name="Message",
+        help_text="Enter the Message",
+    )
+
+
+class CustomPeriodicForm(PeriodicTaskForm):
+    individuals = [(indi.id, indi.first_name) for indi in Individual.objects.all()]
+    groups = [(chat.id, chat.title) for chat in Chat.objects.all()]
+
+    args = ChoiceField(
+        choices=[("chat", groups)] + [("individual", individuals)],
```

Comment:
This is cool! I didn't know you could do this.

You can drop the "+" though:
```suggestion
        choices=[("chat", groups), ("individual", individuals)],
```